src/dfmt/main.d

```diff
@@ -5,7 +5,25 @@
 
 module dfmt.main;
 
-private enum VERSION = "0.5.0";
+debug
+{
+    private enum DEBUG_SUFFIX = "-debug";
+}
+else
+{
+    private enum DEBUG_SUFFIX = "";
+}
+
+static if (is(typeof(import("VERSION"))))
+{
+    import std.string : chomp;
+    private enum VERSION = chomp(import("VERSION")) ~ DEBUG_SUFFIX;
+}
+else
+{
+    private enum VERSION = "0.6.x" ~ DEBUG_SUFFIX;
+}
```

Comment:
I think this should fail if `VERSION` can't be found. This will get outdated extremely soon otherwise, what's the point on keeping a version like this?

Reply:
make is not the only build-system used;
The dub path has to work as well; I cannot figure out howto make writing of the version file work on windows.